gulpfile.js

```diff
@@ -299,5 +303,33 @@ gulp.task('copy-icon-font', () => {
     .pipe(gulp.dest(path.join(PATHS.output, 'Icon', 'Assets')));
 });
 
-gulp.task('default', ['template', 'copy-icon-font']);
+gulp.task('generate-icon-names', () => {
```

Comment:
This should really be part of the `template` task imo

Reply:
true dat, I'll move it there